cluster/examples/kubernetes/ceph/object-realm.yaml

```diff
@@ -0,0 +1,14 @@
+#################################################################################################################
+# Create a multisite realm for a collection of object stores
+#  kubectl create -f object-realm.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephObjectStoreRealm
+metadata:
+  name: my-realm
+  namespace: rook-ceph
+spec:
+  rootPoolSize: 1
```

Comment:
Instead of a single size property, let's discuss making it more flexible for other pool properties